app/models/questionable/option.rb

```diff
@@ -1,9 +1,9 @@
 module Questionable
-  class Option < ActiveRecord::Base
+  class Option < ApplicationRecord
     belongs_to :question
 
     default_scope { order('questionable_options.position ASC') }
 
-    validates_presence_of :title, :question_id
+    validates :title, :question_id, presence: true
```

Comment:
Just spotted another place where we could also use the new syntax. 
https://github.com/bespokepost/questionable/blob/c0099759813d746d4569f2364ff2b0739b85d960/app/models/questionable/date_answer.rb#L7